code/modules/mob/mob_defines.dm

```diff
@@ -106,6 +106,7 @@
 	var/candrop = 1
 	var/lastpuke = 0
 	var/unacidable = 0
+	var/nospells = 0 //Can't use spells. At all.
```

Comment:
If the variable really is that big a deal, yes, I could just put unremovable muzzles on everyone for the 20 seconds